### PR TITLE
Add guideline on closing a PR

### PR DESCRIPTION
**Pull request by fayepal**
Added the guideline that PRs with no activity for at least 14 days will be closed.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ this:
  3. **Commit** changes to your own feature branch
  4. **Push** your work back up to your forked repository
  5. Submit a **Pull request** (PR) so that we can review your changes
+ 
+ ⚠ Pull requests without any activity for at least 14 days will be closed. To avoid this, please reply when asked for more information or update your PR in case in didn't meet the requirements (e.g. failed checks).
```
Comment (LarsMichelsen) [inline CONTRIBUTING.md]: Please be consistent in the line length with the rest of the file. I think the maximum length is ~100 characters. A possible exception are URLs or other language constructs which have to be in a single line.

**Comment by rawiriblundell** [comment]
Request for clarification:  Is this for PR's that have failed testing, or all PR's?

**Comment by svenpanne** [comment]
> Request for clarification: Is this for PR's that have failed testing, or all PR's?

I think we should clarify that this statement is about missing activity on the contributor's side after being asked to change something. It just wouldn't make any sense to auto-close PRs if the we (= the project's team) are too slow responding to it or we haven't decided how/when to merge the PR or parts of it.

So #28 or #116 are definitely not in danger of being autoclosed... :wink:

**Comment by LarsMichelsen** [comment]
Merged, thanks!

**Pull request closed without merging**